biomage/config.yaml

```diff
@@ -7,5 +8,11 @@ source-buckets:
   - "biomage-source-production"
   - "cell-sets-production"
 
+user-pool-id-production: " eu-west-1_ZmmBYI6jO"
+user-pool-id-staging: "eu-west-1_sVQL15Yxu"
```

Comment:
Suggestion, can you not get those values programmatically - see https://github.com/biomage-ltd/ui/blob/develop/src/utils/ssr/getAuthenticationInfo.js - if it's not a big hassle though doing it in python

Reply:
Removed as it's contained in `constants.py`